rules/no-directive-replace.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * disallow the deprecated directive replace property
+ *
+ * This rule disallows the replace attribute in a directive definition object.
+ * The replace property of a directive definition object is deprecated since angular 1.3 ([latest angular docs](https://docs.angularjs.org/api/ng/service/$compile).
+ *
+ * The option `ignoreReplaceFalse` let you ignore directive definitions with replace set to false.
+ *
+ * @version 0.15.0
+ * @category deprecatedAngularFeature
+ */
+'use strict';
+
+module.exports = function(context) {
+    var utils = require('./utils/utils');
+
+    var options = context.options[0] || {};
+    var ignoreReplaceFalse = !!options.ignoreReplaceFalse;
+
+    function isDirectiveDefinitionFunction(fnExpression) {
+        return fnExpression.parent.type === 'CallExpression' && utils.isAngularDirectiveDeclaration(fnExpression.parent);
+    }
+
+    function inDirectiveBody(node) {
+        var block = utils.findNodeTypeInParents(node, 'BlockStatement');
+        return block && isDirectiveDefinitionFunction(block.parent);
+    }
+
+    var reportedNodesByName = {};
+
+    function addPotentialReplaceNode(variableName, node) {
+        var nodeList = reportedNodesByName[variableName] || [];
+
+        var report = {
+            name: variableName,
+            node: node,
+            block: utils.findNodeTypeInParents(node, 'BlockStatement')
```

Comment:
Yes, you are right. BTW Do you know some good docs about all the context methods.